### ISSUE TITLE
Question about Evaluator training

### ISSUE THREAD

**Issue by jidongkuang** [neu-vi/ACMDM#3]
Dear authors,

I'm referencing the T2M code to train an Evaluator using absolute coordinates, but the Top-1 accuracy only reaches 40%, which is significantly lower than the 50.3% reported in your paper. Could you please share how you trained the Evaluator? Additionally, would it be possible to share the relevant model weights?

Looking forward to your guidance! Thank you.

**Comment by cr8br0ze** [comment]
Yes, that's correct. We actually just input the converted one-frame-less motion into the evaluator. This operation will disadvantage our method because of the loss of one frame and global-> relative conversion, on the metric level, but even with this minor disadvantage, ACMDM can still work pretty fine compared to baseline methods, and on different tasks.

We did not really train an absolute coordinate evaluator because (1), this way, it's really easier and fair to compare to baseline methods, and (2), previously (like October 2024) we tried to train T2M evaluator, but with per-joints' xyz absolute coordinate (22x3 -> flatten to 66 -> Z-Norm > evaluator) this way, and it seems like the evaluator have no sense of jitering and smoothness (i.e., even if a motion is terribly jitering it may still be flagged as perfect similar to the original T2M evaluator). But the relative ones from [MARDM](https://github.com/neu-vi/MARDM) seem to be better in sensing jittering and smoothness. So then we just kept using the previous evaluator instead of retraining one.

**Comment by exitudio** [comment]
Thank you for your insight.

**Comment by kingchurch** [comment]
@exitudio  Just curious, why didn't you simply re-train MARDM with the absolute coordinate version of the training data ? What's the advantage of ACMDM over MARDM retrained with AC data ?
